Communication Window for v1

davewiththenicehat opened the pull request:
Completed communications window for v1.
Resolved resizing issue.
Still items for future release. Left notes at top of script.
Set the default size of the right panel to 1/4 rather than 1/8. Hopefully an easy fix for the low res map issue for now.

davewiththenicehat committed:
Communication Window for v1

Completed communications window for v1.
Resolved resizing issue.
Still items for future release. Left notes at top of script.
Set the default size of the right panel to 1/4 rather than 1/8. Hopefully an easy fix for the low res map issue for now.

davewiththenicehat committed:
minor fix

menu.footerlower() not needed

johnnymo87 reviewed (approved):
Those tabs stay the same size, no matter how large I resize the pane, nice work! :trophy: 

![Screenshot from 2019-09-22 09-26-10](https://user-images.githubusercontent.com/3466499/65388578-40a63280-dd1b-11e9-8d95-80521f4b9ca8.png)

BTW, not related to this PR but I thought I'd point it out -- the width of gray border on the left side of the pane seems to offset it by a few pixels, pushing its right side to the right by a few pixels, leaving the pane misaligned with the pain below it.

![Screenshot from 2019-09-22 09-32-28](https://user-images.githubusercontent.com/3466499/65388662-fc676200-dd1b-11e9-855e-3c69682dbef3.png)

davewiththenicehat committed:
bugfix Scroll Bar shift @ Low Res

Resolved scroll bar shifting positions at dissimular resolutions.
Found that I had created a static style sheet for footer. Set it back to the label theme.

davewiththenicehat commented:
Could you try out this code, and verify it works at lower res screens?
Worked while I resized it to a very small size.
After that I'll merge it and consider it ready for v1.
